app/models/concerns/thredded/post_common.rb

```diff
@@ -54,9 +54,31 @@ def context_options
       {
         asset_root: Rails.application.config.action_controller.asset_host || '',
         post:       self,
+        whitelist:  sanitize_whitelist,
       }
     end
 
+    def sanitize_whitelist
```

Comment:
Great points. I've addressed these here - dd0c02e580fdf763628541a0a48c53ce8c7f6ba5

What do you think?